### PR TITLE
[Spells] Implement PVP resist and duration overrides

### PR DESCRIPTION
**Pull request by mackal**
Make use of the spell fields. If you don't want these enabled, I recommend just setting the data equal to normal rather than bothering with a rule.

### CODE REVIEW

File: common/repositories/base/base_spells_new_repository.h [hunk 1]
```diff
@@ -199,8 +199,8 @@ class BaseSpellsNewRepository {
 		int         pvpresistcalc;
 		int         pvpresistcap;
 		int         spell_category;
-		int         field181;
```
Comment (Akkadius) [inline common/repositories/base/base_spells_new_repository.h:202]: Did you manually update these repositories or did you use the script to update ?

**Review by Kinglykrab — APPROVED** [review]

File: common/repositories/base/base_spells_new_repository.h (hunk 1 above)
Comment (mackal) [reply common/repositories/base/base_spells_new_repository.h:202]: The script is unusable for me so I manually updated it.

Comment (Akkadius) [reply common/repositories/base/base_spells_new_repository.h:202]: What do you mean, unusable?

Comment (Akkadius) [reply common/repositories/base/base_spells_new_repository.h:202]: How to use is here: https://eqemu.gitbook.io/server/in-development/developer-area/repositories

```
perl ~/code/utils/scripts/generators/repository-generator.pl ~/server spells_new
```
Checked out your code and looks like you got it all manually 100% anyways

**Review by Akkadius — APPROVED** [review]

**Pull request merged**